src/NzbDrone.Core/Indexers/RssSyncService.cs

```diff
@@ -60,15 +67,70 @@ private ProcessedDecisions Sync()
             return processed;
         }
 
+        private IEnumerable<ReleaseInfo> FilterByIndexer(IEnumerable<ReleaseInfo> releases)
+        {
+            var indexerBackOff = new Dictionary<int, bool>();
+
+            foreach (var release in releases)
+            {
+                bool ignore;
+                if (!indexerBackOff.TryGetValue(release.IndexerId, out ignore))
+                {
+                    var indexerStatus = _indexerStatusService.GetIndexerStatus(release.IndexerId);
+
+                    if (indexerStatus == null || !indexerStatus.DisabledTill.HasValue || indexerStatus.DisabledTill.Value < DateTime.UtcNow)
+                    {
+                        indexerBackOff[release.IndexerId] = ignore = false;
+                    }
+                    else
+                    {
+                        indexerBackOff[release.IndexerId] = ignore = true;
+                    }
+                }
+
+                if (!ignore)
+                {
+                    yield return release;
+                }
+            }
+        }
+
         public void Execute(RssSyncCommand message)
         {
+            var rssStarted = DateTime.UtcNow;
+
             var processed = Sync();
             var grabbedOrPending = processed.Grabbed.Concat(processed.Pending).ToList();
 
             if (message.LastExecutionTime.HasValue && DateTime.UtcNow.Subtract(message.LastExecutionTime.Value).TotalHours > 3)
             {
-                _logger.Info("RSS Sync hasn't run since: {0}. Searching for any missing episodes since then.", message.LastExecutionTime.Value);
-                _episodeSearchService.MissingEpisodesAiredAfter(message.LastExecutionTime.Value.AddDays(-1), grabbedOrPending.SelectMany(d => d.RemoteEpisode.Episodes).Select(e => e.Id));
+                var lastGap = rssStarted;
```

Comment:
to be honest, I don't think this should run as part of RSS sync. it should be it's own command.